main.py

```diff
@@ -1,10 +1,63 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+        for player_nickname, player_data in data.items():
+
+            # RACE
+
+            if not Race.objects.filter(
+                    name=player_data["race"]["name"]).exists():
+                player_race = Race.objects.create(
+                    name=player_data["race"]["name"],
+                    description=player_data["race"]["description"]
+                )
+            else:
+                player_race = Race.objects.get(
+                    name=player_data["race"]["name"])
```

Comment:
This formatting is not considered ideal here. If you start a new line after the opening parenthesis, you should also do the same before the closing one. Same at $17, $40, $47.

Reply:
yep!

Reply:
Just so you know: so far you fixed it only at line 47. :)